elisp-bytecode.texi

```diff
@@ -537,7 +538,38 @@ opcode, in little-endian byte order.
 @unnumberedsubsec @code{stack-ref} (1--7)
 @kindex stack-ref
 
-A stack reference
+A stack reference. This is used only in lexical binding.
+
+Added in Emacs 24.1
+
+@subsubsection Warning
+Running an instruction with opcode 0 (logically this would be called
+@code{stack-ref[0]}), will cause an immediate abort of Emacs!
+
+0 is typically an invalid bytecode and assembly instructions because 0
```

Comment:
It's an invalid opcode, but a common and valid byte in a bytecode string. I think that's worth pointing out.

Reply:
If you have specific wording, let me know. I'm not sure what would change.

Turns out in 25.1 the behviour change prints an error and doesn't abort. I'll push the updates to that soon.

Reply:
I'd just change "invalid bytecode and assembly instructions" to "invalid opcode".